pkg/registry/clusterrolebinding/policybased/storage.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Should there be a similar check on deletes to prevent stripping permissions from others? E.g. if someone gave me a 'read-only' in a namespace and an ability to manage RoleBindings in that namespace, I probably shouldn't be able to remove a RoleBinding with an 'admin' roleRef. There is a validation to prevent roleRef change on update, but I can't find anything preventing deletes.